lib/manager/git-submodules/readme.md

```diff
@@ -1 +1,4 @@
-Keeps Git submodules updated within a repository.
+Keeps publicly accessible Git submodules updated within a repository.
+
+Renovate does not support updating Git submodules that are hosted on a private repository.
+Subscribe to issue #... on GitHub to keep track of our progress towards supporting private Git submodules.
```

Comment:
I'll need a issue number that people can subscribe to to track progress towards this feature. I know we have a lot of questions for this feature, but I'm not sure _which_ issue people should be directed to follow.

Reply:
#10369 and #10149 could be the issues we want end-users to track?

Reply:
#10149

Reply:
```suggestion
Subscribe to [issue #10149 on GitHub](https://github.com/renovatebot/renovate/issues/10149) to keep track of our progress towards supporting private Git submodules.
```